crowbar_framework/app/models/proposal_queue.rb

```diff
@@ -0,0 +1,35 @@
+class ProposalQueue < ActiveRecord::Base
+  # FIXME: remove this when the export is properly implemented
+  class_attribute :chef_type
+
+  self.chef_type = "data_bag_item"
+  # only accept one default record named "queue"
+  self.primary_key = "name"
+
+  serialize :properties, Utils::JSONWithIndifferentAccess
+
+  def proposals
+    ProposalQueue.find("queue").properties["proposal_queue"]
+  end
+
+  def <<(item)
+    pq = ProposalQueue.find("queue")
```

Comment:
@aspiers we could also use a private method to just return an ActiveRecord object to get rid of the class identifier dups.

```ruby
...
private
  def proposal_queue
    ProposalQueue.find("queue")
  end
```

but that is a matter of taste I guess. anyone who has an opinion please share.